CONTRIBUTING.md

```diff
@@ -0,0 +1,223 @@
+
+This section guides you through submitting a bug report for Metafacture. Following these guidelines helps maintainers and the community understand your report, reproduce the behavior and find related reports.
+
+Before creating bug reports, please check if an issue with this bug [already exists](https://github.com/metafacture/metafacture-core/issues). When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-bug-report).
+
+**Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### How Do I Submit A Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create a [new issue on the repository](https://github.com/metafacture/metafacture-core/issues/new).
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the exact steps which reproduce the problem** in as many details as possible. For example, start by explaining how you are running Metafacture (e.g. running flux.sh/flux.bat or as a Java library). When listing steps, **don't just say what you did, but explain how you did it**. For example, if you're describing the behavior of a specific Morph/Fix function, provide the actual workflow (Flux/Java) and the full Morph file you're using (see also next point). 
+* **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code). The ideal example would be a stripped-down runnable use case using some sample data showing the problematic behavior.
+* **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+* **Explain which behavior you expected to see instead and why.**
+* **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Metafacture, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
+
+Before creating enhancement suggestions, please **perform a [cursory search](https://github.com/search?q=is%3Aissue+repo%3Ametafacture%2Fmetafacture-core)** to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one. When you are creating an enhancement suggestion, please include as many details as possible:
```

Comment:
Sometimes it's spoken about "Metafacture" and sometimes about "Metafacture-core". I know this file resides in the metafacture-core repo, so it would be intuitive to just refer to that. But, e.g. it may occur to a user that some functionality is missing, say something that is already covered by [metafacture-csv-plugin ](https://github.com/metafacture/metafacture-csv-plugin). So I suggest to stick to "Metafacture" as a whole, (also there is no issue in bespoken metafacture-csv-plugin repo), using https://github.com/search?q=org%3Ametafacture&type=issues ... ah, hm, no this gets also a lot of metafacture-playground issues, which isn't (?!) what you really want . So, I don't know how to make aware of these plugins. Maybe with:
```suggestion
Before creating enhancement suggestions, please **perform a [cursory search](https://github.com/search?q=is%3Aissue+repo%3Ametafacture%2Fmetafacture-core)** to see if the enhancement has already been suggested. Also have a look at the [metafacture-plugins](https://github.com/search?q=org%3Ametafacture). If it has already been suggested, add a comment to the existing issue instead of opening a new one. When you are creating an enhancement suggestion, please include as many details as possible:
```

Reply:
> ah, hm, no this gets also a lot of metafacture-playground issues, which isn't (?!) what you really want .

You can also exclude repos in the search with `-`, e.g. https://github.com/search?q=org%3Ametafacture+-repo%3Ametafacture%2Fmetafacture-fix+-repo%3Ametafacture%2Fmetafacture-playground&type=issues. However, I am not sure this is the best solution here.